sources/packages/web/src/components/common/StudentOverawardDetails.vue

```diff
@@ -1,54 +1,58 @@
 <template>
   <v-card class="mb-5">
```

Comment:
Same from https://github.com/bcgov/SIMS/pull/1816/files#r1139577805

Reply:
Fixed it.